### DISCUSSION TITLE
Include contact assignments in api requests

### DISCUSSION THREAD

**Discussion by peterbaumert** [netbox-community/netbox#9332] [Ideas]
Currently when I assign a contact to a device/site or whatever it will not be included in the api requests of
- api/dcim/devices/
- api/dcim/sites/
- and so on

For me this would be very beneficial in all the automation tools. e.g. at contact to SNMP config.

I didn't want to open a feature request yet to hear first if I'm alone with this or it could be good for others too.

KR
Pete

**Comment by gabrielat** [comment]
I would like that to be implemented too.
Also, it would be nice if the Contact Assignment's API showed all Contact infos, besides "Name" and "Description". I need it to show "Link" and "Custom Fields" at least.
